io-sim/io-sim.cabal

```diff
@@ -26,18 +26,8 @@ source-repository head
 library
   hs-source-dirs:      src
   exposed-modules:     Control.Monad.IOSim
+  other-modules:       Control.Monad.IOSim.Internal
```

Comment:
What's the reason for introducing `Control.Monad.IOSim.Internal`?

Note that many libraries still expose internal modules so it's there when you need it, but without stability guarantees.

Reply:
I haven't change what is exposed.  `IOSim` is not extensible in any way so I don't see a benefit for exposing its all internals.  However, I wanted to split the package into api / implementation, making it easier for those who'd like to learn the details of the library.

Reply:
Ok